Formula/bumpversion.rb

```diff
@@ -0,0 +1,27 @@
+class Bumpversion < Formula
+  include Language::Python::Virtualenv
+
+  desc "Version-bump your software with a single command"
+  homepage "https://pypi.python.org/pypi/bumpversion"
+  url "https://pypi.io/packages/source/b/bumpversion/bumpversion-0.5.0.tar.gz"
```

Comment:
https://pypi.org/project/bumpversion/ has `0.5.3` but 
https://github.com/peritus/bumpversion/releases has `0.5.0` as latest.

`0.5.0` is almost four years old and `0.5.3` is three years old. Also no upstream commits for almost 3 years.

Reply:
Good catch.  GitHub has those tagged also, just looks like peritus didn't make a "GitHub Release(tm)" out of it.

I'll update these tomorrow.

I'm not worried about the lack of upstream work, it's been a stable tool, very useful for what it does and I don't know of any alternatives in the space.